spec/index.bs

```diff
@@ -1128,6 +1128,8 @@ dictionary GPULimits {
     GPUSize32 maxStorageTexturesPerShaderStage = 4;
     GPUSize32 maxUniformBuffersPerShaderStage = 12;
     GPUSize32 maxUniformBufferBindingSize = 16384;
+    GPUSize64 sampleCountMask = 5;
+    GPUSize64 maxTexelBlockSize = 64;
```

Comment:
It would be good to have confirmation from the D3D12 and Metal team that this is the only additional restriction we can see in the future. Same thing for the Vulkan WG.

Reply:
Yesterday I suggested describing the limit this way in prose (or just not at all for now), but not exposing API surface for it in GPULimits. It doesn't serve any purpose until we have 8x MSAA (the largest texel block size we have right now is 4x rgba32float = 64 bytes). That way we can change how we describe it if needed.

Reply:
It's certainly not the only restriction. I need to see how to fit Metal in there...